controllers/alerts/alerts.go

```diff
@@ -215,3 +217,18 @@ func createOperatorHealthStatusRule() monitoringv1.Rule {
 		Expr:   intstr.FromString(`label_replace(vector(2) and on() ((kubevirt_hco_system_health_status>1) or (count(ALERTS{kubernetes_operator_part_of="kubevirt", alertstate="firing", operator_health_impact="critical"})>0)) or (vector(1) and on() ((kubevirt_hco_system_health_status==1) or (count(ALERTS{kubernetes_operator_part_of="kubevirt", alertstate="firing", operator_health_impact="warning"})>0))) or vector(0),"name","kubevirt-hyperconverged","","")`),
 	}
 }
+
+func createSingleStackIPv6AlertRule() monitoringv1.Rule {
+	return monitoringv1.Rule{
+		Alert: singleStackIPv6Alert,
+		Expr:  intstr.FromString("kubevirt_hco_hyperconverged_single_stack_ipv6 == 0"),
```

Comment:
> 	SingleStackIPv6True  = float64(1)
	SingleStackIPv6False = float64(0)
	
shouldn't this be `kubevirt_hco_hyperconverged_single_stack_ipv6 == 1`?